cfy_manager/components/validations.py

```diff
@@ -328,13 +337,30 @@ def _check_key_and_cert(prefix):
         key_modulus = sudo(key_modulus_command).aggr_stdout.strip()
         cert_modulus = sudo(cert_modulus_command).aggr_stdout.strip()
         if cert_modulus != key_modulus:
-            raise ValidationError('Key {0} ({1}) does not match the '
-                                  'cert {2} ({3})'
-                                  .format(key_filename, key_input,
-                                          cert_filename, cert_input))
+            raise ValidationError('Key {0} ({1}.{2}) does not match the '
```

Comment:
Maybe change '.' to ':' since it's YAML and would make more sense to the user?

Reply:
Hmm... I thought we were standardised on dot separating the paths elsewhere? Are we using : much anywhere else?

Reply:
I'm actually not aware of that, but it just made sense to me since this is a YAML key:value and not key.value